src/hb-subset-glyf.cc

```diff
@@ -214,7 +213,7 @@ _write_glyf_and_loca_prime (hb_subset_plan_t              *plan,
                                             loca_prime_size);
     _update_components (plan, glyf_prime_data_next, length);
 
-    glyf_prime_data_next += length;
+    glyf_prime_data_next += length + (length % 2); // Align to 2 bytes for short loca.
```

Comment:
Do we always do this, or only if using short loca? It's fine if we always do it for now. Just add a TODO item to not doing for long loca, since that would be more compact.

Reply:
Currently we always do it. I've added a TODO.